sklearn/ensemble/tests/test_stacking.py

```diff
@@ -859,3 +859,35 @@ def test_stacking_classifier_base_regressor():
     clf.predict(X_test)
     clf.predict_proba(X_test)
     assert clf.score(X_test, y_test) > 0.8
+
+
+def test_stacking_final_estimator_attribute_error():
+    """Check that we raise the proper AttributeError when the final estimator
+    does not implement the `decision_function` method, which is decorated with
+    `available_if`.
+
+    Non-regression test for:
+    https://github.com/scikit-learn/scikit-learn/issues/28108
+    """
+    X, y = load_breast_cancer(return_X_y=True)
+    X_train, X_test, y_train, _ = train_test_split(
+        scale(X), y, stratify=y, random_state=42
+    )
```

Comment:
we can probably use a smaller dataset, and avoid calling `scale` to make the test faster.

Reply:
The scale would avoid a `ConvergenceWarning` of the `LogisticRegression` certainly. To be checked if we remove it.

Reply:
we can do `make_classification` instead then. Would be faster. The data doesn't matter here.

Reply:
I now put `make_classifiction` for data creation.